cli/js/files.ts

```diff
@@ -215,8 +225,10 @@ export function seekSync(rid: number, offset: number, whence: SeekMode): void {
 
 /** Seek a file ID to the given offset under mode given by `whence`.
  *
- *      const file = await Deno.open("/foo/bar.txt");
- *      await Deno.seek(file.rid, 0, 0);
+ *      (async () => {
+ *        const file = await Deno.open("/foo/bar.txt");
+ *        await Deno.seek(file.rid, 0, 0);
+ *      })();
```

Comment:
This isn't necessary, if you see await you should know async. Especially with TLA support.

Reply:
I thought so too. But I was copying over changes to lib.deno.ns.d.t introduced in #4170.

Reply:
Hmm, I can't find a change that adds an async wrapper there.

Reply:
Whoops, you're right. The async() wasn't introduced then. It's been in lib.deno for a while. It was just the process of synchronizing the comments in lib.deno with those in the source files, which was motivated by #4170, that introduced that change to cli/js/files.ts.

I'd be happy to remove the async() => from both places.